system/Config/Services.php

```diff
@@ -693,19 +694,19 @@ public static function router(RouteCollectionInterface $routes = null, Request $
 	 * The Security class provides a few handy tools for keeping the site
 	 * secure, most notably the CSRF protection tools.
 	 *
-	 * @param App|null $config
-	 * @param boolean  $getShared
+	 * @param \Config\Security|null $config
+	 * @param boolean               $getShared
 	 *
 	 * @return Security
 	 */
-	public static function security(App $config = null, bool $getShared = true)
+	public static function security(SecurityConfig $config = null, bool $getShared = true)
```

Comment:
it is bc break, please read https://github.com/codeigniter4/CodeIgniter4/blob/develop/CONTRIBUTING.md#breaking-changes